docs/iris/src/whatsnew/contributions_2.0/incompatiblechange_2017-Oct-23_cell-datetime-objs.txt

```diff
@@ -0,0 +1,9 @@
+:class:`iris.coords.Cell` objects in time coordinates now contain datetime objects by default and not numbers.
+For example::
+    >>> cube = iris.load_cube(iris.sample_data_path('air_temp.pp'))
+    >>> print(cube.coord('time').cell(0).point)
+        1998-12-01 00:00:00
+
+This change particularly impacts constraining datasets on time. All time constraints
+must now be constructed with datetime objects or :class:`~iris.time.PartialDateTime` objects.
+See userguide section 2.2.1 for more details on producing time constraints.
```

Comment:
Nice.  I approve of the detail and definitiveness (I know there is a proper word that belongs there, but I can't think of what it would be) of this what's new entry.